src/Tizen.Security.PrivacyPrivilegeManager/Tizen.Security/PrivacyPrivilegeManager.cs

```diff
@@ -219,7 +429,7 @@ internal ResponseContext(string privilege)
 
             internal void FireEvent(CallCause _cause, RequestResult _result)
             {
-                _ResponseFetched?.Invoke(null, new RequestResponseEventArgs() { cause = _cause, result = _result, privilege = _privilege });
+                _ResponseFetched?.Invoke(null, new RequestResponseEventArgs { cause = _cause, result = _result, privilege = _privilege });
```

Comment:
Why sender was null? it should be `this`(ResponseContext)

Reply:
This  bug was introduced in first Tizen.Security.PrivacyPrivilegeManager API implementation.
I fixed it now.